src/python/pants/bin/local_pants_runner.py

```diff
@@ -202,6 +202,7 @@ def __init__(self, build_root, exiter, options, options_bootstrapper, build_conf
     self._build_config = build_config
     self._target_roots = target_roots
     self._graph_session = graph_session
+    self._scheduler = scheduler
```

Comment:
I think this isn't actually a `Scheduler`, but is instead a `SchedulerSession`, so this should maybe be `self._scheduler_session = scheduler_session` rather than `this._scheduler = scheduler`?